packages/shopware-6-client/src/interfaces/models/common/Entity.ts

```diff
@@ -0,0 +1,7 @@
+export interface Entity {
+  _uniqueIdentifier: string;
+  versionId: string;
+  translated: [];
```

Comment:
Which type of Array do we have here?

Reply:
Just an array in Shopware. No specific type.